lib/diagnostic.rb

```diff
@@ -6,21 +6,22 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+
+Diagnostic.script = "ruby lib/example.rb"
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = "pry"
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = "star wars episode viii"
```

Comment:
Ruby uses "snake case" for its naming convention so this should be `star_wars_episode_viii`.